solutions/usaco-811.mdx

```diff
@@ -229,4 +229,38 @@ public class snowboot {
 ```
 
 </JavaSection>
+
+<PySection>
+
+```python
+with open('snowboots.in') as r:
+	n, b = map(int, r.readline().split())
+	depth = list(map(int, r.readline().split()))
+	max_depth = [[*map(int, r.readline().split())] for _ in range(b)]
+
+stor = [[0] * b for _ in range(n)]
```

Comment:
better name pls

Reply:
stor as in storage?

Reply:
or are you talking about depth and max_depth?

Reply:
`stor`
bc like what's it storing?

Reply:
so should i change it to dp?

Reply:
no like the point is that name isn't really informative about what it's storing

Reply:
oh okay i'll change it.

Reply:
bump